src/components/svg/index.tsx

```diff
@@ -33,7 +33,7 @@ export class SmoothlySvg {
 					"min-width": width,
 					"--size-height": height,
 					"--size-width": width,
-					"--color": this.color,
+					"--text-color": this.color,
```

Comment:
is this intended?

Reply:
No.

Reverting the file to its original state